src/app/app-routing.module.ts

```diff
@@ -51,6 +52,14 @@ export const routes: Routes = [
   },
   // Profile
   {
+    path: '_profile',
+    resolve: {
+      profileNavigation: ProfileResolver
```

Comment:
Where is `profileNavigation` defined? Or is it not needed since it is just a JSON object?

Reply:
It's not used by the target since the resolver redirects, it is just a json object but you need to have a resolver defined that is named.

Reply:
thanks, just wanted to confirm